packages/components/src/internal/components/domainproperties/dataset/models.ts

```diff
@@ -171,10 +169,8 @@ export class DatasetModel implements IDatasetModel {
         }
     }
 
-    getDomainKind(): string {
-        return getServerContext().moduleContext.study.timepointType === 'VISIT'
-            ? 'StudyDatasetVisit'
-            : 'StudyDatasetDate';
+    getDomainKind(timepointType: string): string {
+        return timepointType === 'VISIT' ? 'StudyDatasetVisit' : 'StudyDatasetDate';
```

Comment:
Might be useful for `VISIT` to be a constant.

Reply:
done